.circleci/config.yml

```diff
@@ -131,7 +131,7 @@ workflows:
       - cypress/run:
           name: 'UI Tests - Chrome'
           browser: chrome
-          spec: cypress/integration/*
+          spec: cypress/e2e/*
```

Comment:
I expected changing `spec` without also changing the location of the E2E tests would break the build, but it didn't, probably because we have a custom `command` defined and the project [cypress.json](https://github.com/cypress-io/cypress-documentation/blob/master/cypress.json) `testFiles` is set to `**/*.spec.js`.

Can you try removing this `spec` line and see if it still builds? If this config isn't used, we should remove it.